games/ludo/master.py

```diff
@@ -92,169 +98,140 @@ def play(self) -> None:
         valid, the player is reprompted up to a maximum of three times, after
         which time, the game is aborted.
         """
-
         while not self._check_game_status():
             logger.info("Game turn: %d", self.game.turn)
             self.log_next_turn()
-            # log current state
-            action = {'type': f'current state', 'content': self.game.current_state}
-            self.log_event(from_="GM", to="GM", action=action)
+            self.log_event(
+                from_="GM",
+                to="GM",
+                action={
+                    'type': 'current state',
+                    'content': self.game.current_state
+                }
+            )
             logger.info(f"current_state: {self.game.current_state}")
-            for i, player in enumerate(self.players_dic.keys()):
-                if len(self.players_dic.keys()) > 1:
-                    # rolls needs to be format list[(roll_player_1, roll_player_2),..]
-                    roll: int = self.game.rolls[self.game.turn][i]
-                else:
-                    roll: int = self.game.rolls[self.game.turn]
+            
+            for index, player in enumerate(self.players_dic.keys()):
```

Comment:
Serious question, why are you trying to compress everything into single-line statements, it doesn't improve readability by default